target/linux/realtek/files-5.10/drivers/clk/realtek/clk-rtl83xx.h

```diff
@@ -0,0 +1,87 @@
+// SPDX-License-Identifier: GPL-2.0-only
+/*
+ * Realtek RTL83XX clock headers
+ * Copyright (C) 2022 Markus Stockhausen
+ */
+
+/*
+ * Several stuff
+ */
+
+#define RTL_SRAM_MARKER				(0x5EAF00D5)
+#define RTL_SRAM_BASE				(0x9F000000)
+
+/*
+ * Switch registers
+ */
+
+#define RTL_SW_CORE_BASE			(0xBB000000)
+
+#define RTL838X_PLL_GLB_CTRL			(0x0FC0)
+#define RTL838X_PLL_CPU_CTRL0			(0x0FC4)
+#define RTL838X_PLL_CPU_CTRL1			(0x0FC8)
+#define RTL838X_PLL_LXB_CTRL0			(0x0FD0)
+#define RTL838X_PLL_LXB_CTRL1			(0x0FD4)
+#define RTL838X_PLL_MEM_CTRL0			(0x0FDC)
+#define RTL838X_PLL_MEM_CTRL1			(0x0FE0)
+
+#define RTL839X_PLL_GLB_CTRL			(0x0024)
+#define RTL839X_PLL_CPU_CTRL0			(0x0028)
+#define RTL839X_PLL_CPU_CTRL1			(0x002C)
+#define RTL839X_PLL_LXB_CTRL0			(0x0038)
+#define RTL839X_PLL_LXB_CTRL1			(0x003C)
+#define RTL839X_PLL_MEM_CTRL0			(0x0048)
+#define RTL839X_PLL_MEM_CTRL1			(0x004C)
+
+#define RTL_PLL_CTRL0_CMU_SEL_PREDIV(v)		(((v) >> 0) & 0x3)
+#define RTL_PLL_CTRL0_CMU_SEL_DIV4(v)		(((v) >> 2) & 0x1)
+#define RTL_PLL_CTRL0_CMU_NCODE_IN(v)		(((v) >> 4) & 0xFF)
```

Comment:
(yes, GENMASK(7, 0) is okay as is GENMASK(32, 0); it makes your intent very clear without magic values :)